### PR TITLE
Add some nice unicode art

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -68,9 +68,14 @@ function getInsult(message, user) {
     'Meine Großmutter kann besser Rust als du Wicht, ' + u,
     'Du bist das hässlichste Wesen dass ich jemals sah, ' + u,
     'Gibt es eine größere Memme als dich, ' + u + '?',
-    
+
     // Not inspired by monkey island.
     'Dini Mueter.',
+
+	// some nice Unicode art
```
Comment (netzhuffle) [inline bot.js]: Please keep the code style consistent, use spaces instead of tabs.

File: bot.js [hunk 2]
```diff
@@ -68,9 +68,14 @@ function getInsult(message, user) {
...
+	'凸(-_-)凸',
+	'╭∩╮(-_-)╭∩╮',
+	'(╯°□°）╯︵ ┻━┻',
```
Comment (schmijos) [inline bot.js]: Wunderhübsch ;-)

File: bot.js (hunk 1 above)
Comment (schmijos) [inline bot.js]: Travis with Linting?

Comment (rnestler) [inline bot.js]: ah sorry, I should configure my editor...

**Comment by rnestler** [comment]
OK I fixed the whitespace issue and squashed it into the commit.

**Pull request merged**